stackit/internal/services/dns/recordset/resource_test.go

```diff
@@ -100,7 +102,8 @@ func TestMapFields(t *testing.T) {
 				Active:      types.BoolNull(),
 				Comment:     types.StringNull(),
 				Error:       types.StringNull(),
-				Name:        types.StringValue("name"),
+				Name:        types.StringNull(),
+				FQDN:        types.StringValue("name"),
```

Comment:
it would be nice to check also that if there is a name in the model, `mapFields` doesn't overwrite it with what comes from the API

You can do that by setting some `Name` (different than the one in the API response) on the `state` before calling the function and then checking that it remains the same